code/modules/admin/admin_verbs.dm

```diff
@@ -322,10 +322,9 @@ var/list/admin_verbs_hideable = list(
 	if(istype(mob,/mob/dead/observer))
 		//re-enter
 		var/mob/dead/observer/ghost = mob
-		if (!ghost.can_reenter_corpse)
+		if (ghost.can_reenter_corpse)
 			log_admin("[key_name(usr)] re-entered corpse")
 			message_admins("[key_name_admin(usr)] re-entered corpse")
-		ghost.can_reenter_corpse = 1			//just in-case.
```

Comment:
aghost is suppose to be a more powerful version of ghost, it SHOULD imply everything normal ghosting has.

No need to make admins use two different verbs.